src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html

```diff
@@ -0,0 +1,240 @@
+              <a class="dropdown-item" (click)="moveToFolder(mailFolderTypes.INBOX)" translate="mail_sidebar.inbox">
+                <i class="icon icon-inbox"></i>Inbox</a>
+              <a class="dropdown-item" (click)="moveToFolder(mailFolderTypes.DRAFT)" translate="mail_sidebar.draft">
+                <i class="icon icon-inbox"></i>Draft</a>
+              <a class="dropdown-item" (click)="moveToFolder(mailFolderTypes.ARCHIVE)" translate="mail_sidebar.archive">
+                <i class="icon icon-archive"></i>Archive</a>
+              <span class="divider-item d-block"></span>
+              <a class="dropdown-item" data-toggle="modal" data-target="#createFoldersModal"
+                 translate="mail_list.create_folder">
+                <i class="icon icon-folder"></i>Create folder</a>
+            </div>
+          </div>
+        </li>
+        <li class="btn-dropdown mailbox-dropdown mailbox-dropdown-lg more-options-dropdown pl-0">
+          <div ngbDropdown class="dropdown">
+            <button ngbDropdownToggle class="disabledx" role="button" id="moreOptionsDropdownMenuButton"
+                    translate="mail_list.more">
+              More
+            </button>
+            <div id="moreOptionsDropdownMenu" ngbDropdownMenu aria-labelledby="moreOptionsDropdownMenuButton">
+              <a class="dropdown-item" (click)="markAsStarred()" translate="mail_list.add_star">Add star</a>
+              <a class="dropdown-item" (click)="markAsRead()" translate="mail_list.mark_as_read">Mark as read</a>
+              <a class="dropdown-item" (click)="markAsRead(false)" translate="mail_list.mark_as_unread">Mark as
+                unread</a>
+            </div>
+          </div>
+        </li>
+      </ul>
+    </div>
+
+    <div class="col-sm-4 mailbox-actionbar-col hidden-xs-down">
+      <div class="mail-controls d-flex align-items-center justify-content-end">
+        <div class="mail-result-text text-muted">
+          <small>1-50 of 2346</small>
+        </div>
+        <div class="mail-nav-control">
+          <button role="button" class="mail-nav-control-prev">
+            <i class="icon icon-chevron-left"></i>
+          </button>
+          <button role="button" class="mail-nav-control-next">
+            <i class="icon icon-chevron-right"></i>
+          </button>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="mail-listing">
+  <div class="table-responsive table-hover table-fixed">
+    <table class="table table-inbox">
+      <tbody>
+      <tr class="mail-list-row" *ngFor="let mail of mails; let i=index"
+          [ngClass]="{'is-unread': !mail.read, 'is-read': mail.read, 'is-selected':mail.marked}">
+
+
+        <!-- Check box -->
+        <td class="mail-checkbox pl-4">
+          <div class="fancy-field-group">
+            <input class="d-none fancy-field-control" id={{mail.id}} type="checkbox" [(ngModel)]="mail.marked"/>
```

Comment:
Missing quotes around value of `id`

Reply:
@Waseem826 I just moved this code, this is written by zain.
Secondy I think it is allowed without quotes.